client/state/selectors/is-requesting-jetpack-onboarding-settings.js

```diff
@@ -0,0 +1,29 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { getRequest } from 'state/selectors';
+import { requestJetpackOnboardingSettings } from 'state/jetpack-onboarding/actions';
+
+/**
+ * Returns true if we are currently making a request to fetch the Jetpack settings. False otherwise
```

Comment:
Nit: maybe specify that this is a Jetpack onboarding settings request, not just a Jetpack settings request.

Reply:
It is for now 😬

Reply:
(I'll actually leave this one as-is since it's quite likely I'll miss it in #23393.)